ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-stable-4.17-upgrade-from-stable-4.17.yaml

```diff
@@ -97,8 +97,8 @@ tests:
     test:
     - chain: openshift-upgrade-qe-sanity-disconnected
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-private-s3-with-ep-sts-ec2-elb
-- as: aws-ipi-disc-priv-tp-f28
```

Comment:
We run this job with a OTA test case, so suggest to keep it on f28.

Reply:
done